Docker/test/selenium/wdio.conf.js

```diff
@@ -78,7 +78,15 @@ exports.config = {
 	 * @param {Object} test Mocha Test object
 	 */
 	afterTest: function ( test ) {
-		const filePath = saveScreenshot( process.env.SUITE + '--' + test.title );
-		console.log( '\n\tScreenshot: ' + filePath + '\n' );
+		const screenshotPath = browser.config.screenshotPath;
+		const folder = ( process.env.DATABASE_IMAGE_NAME + '-' + process.env.SUITE ).replace( /[^a-zA-Z_0-9]/g, '_' );
+		browser.config.screenshotPath = screenshotPath + '/' + folder;
+		try {
+			saveScreenshot( test.title );
+		} catch ( error ) {
+			console.error( 'failed writing screenshot ...' );
+			console.error( error );
+		}
+		browser.config.screenshotPath = screenshotPath;
```

Comment:
Why do you reset the `screenshotPath` here?

Reply:
yeah makes no sense.

Reply:
Or no that actually made sense because other wise we are just gonna append the folder to each test run.

```
.
├── mariadb_10_3_elasticsearch
│   ├── mariadb_10_3_elasticsearch
│   │   ├── mariadb_10_3_elasticsearch
│   │   │   ├── mariadb_10_3_elasticsearch
│   │   │   │   ├── mariadb_10_3_elasticsearch
│   │   │   │   │   └── should-be-able-to-search-via-alias.png
│   │   │   │   └── should-be-able-to-search-case-insensitive.png
│   │   │   └── Should-be-able-to-run-ForceSearchIndex.php.png
│   │   └── Should-able-able-to-run-UpdateSearchIndexConfig.php.png
│   └── Should-be-able-to-set-alias.png
└── Should-create-an-item.png

```